README.md

```diff
@@ -101,66 +93,62 @@ The following are the samples.
 
 #### Rosbag
 
-1. Download sample map from [here](https://drive.google.com/open?id=1ovrJcFS5CZ2H51D8xVWNtEvj_oiXW-zk).
-
-2. Download sample rosbag from [here](https://drive.google.com/open?id=1BFcNjIBUVKwupPByATYczv2X4qZtdAeD).
-3. Launch Autoware
+1. Download sample pointcloud and vector maps from [here](https://drive.google.com/open?id=1ovrJcFS5CZ2H51D8xVWNtEvj_oiXW-zk).
+2. Download a sample rosbag from [here](https://drive.google.com/open?id=1BFcNjIBUVKwupPByATYczv2X4qZtdAeD).
+3. Open a terminal and launch Autoware
 
 ```sh
 cd AutowareArchitectureProposal
 source install/setup.bash
 roslaunch autoware_launch logging_simulator.launch map_path:=[path] vehicle_model:=lexus sensor_model:=aip_xx1 rosbag:=true

```

Comment:
Here you could use the convention `/path/to/map_file` instead of `[path]`. If I remember correctly, the pointcloud and vector map needed to be in a folder and the path needed to be specified to that folder. Maybe that can be explicitly specified somewhere.

Reply:
That is correct. In that instance where the full path to a folder is required rather than a file, could I just say `/path/to/map_folder` instead?